posthog/models/session_recording_event/util.py

```diff
@@ -52,6 +57,54 @@ def create_session_recording_event(
     return str(uuid)
 
 
+def bulk_create_session_recording_event(events: List[Dict[str, Any]]) -> None:
+    """
+    Test only
+    """
+    # timestamp = cast_timestamp_or_now(timestamp)
+
+    inserts = []
+    params: Dict[str, Any] = {}
+    for index, event in enumerate(events):
+        timestamp = event["timestamp"]
+        if isinstance(timestamp, str):
+            timestamp = isoparse(timestamp)
+        else:
```

Comment:
I'd suggest `elif isinstance(timestamp, datetime)` here, and then in the `else` block, throw a ValueError saying we got a timestamp type we weren't expecting

it's a common Python idiom, but in general I dislike this sort of loosey-goosey typing where you allow callers to pass in anything that looks timestamp-ish and you try to convert it to the format you want. (insert ["grumble grumble Postel's law bad"](https://datatracker.ietf.org/doc/html/draft-thomson-postel-was-wrong-00) here)

but I'm guessing there's existing callers that expect to be able to pass in either one, so the larger change isn't necessary in the scope of this PR.

Reply:
Hm the else here is a call to `.astimezone` which already throws if it's not a datetime, so this feels duplicative?